extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/CacheResultInterceptor.java

```diff
@@ -47,28 +48,42 @@ public Object intercept(InvocationContext invocationContext) throws Throwable {
                 @Override
                 public Object apply(Object k) {
                     try {
-                        return invocationContext.proceed();
+                        Object invocationResult = invocationContext.proceed();
+                        if (invocationResult instanceof Uni) {
+                            LOGGER.debugf("Adding UncomputedUniValue entry with key [%s] into cache [%s]", key,
```

Comment:
In general, we say `unresolved` and not `Uncomputed`, but that's for purist :-D

Reply:
(resolve/unresolved comes from the promise abstraction)

Reply:
I was actually not really happy with that `Uncomputed` name. Thanks for suggesting a better one.